.github/CODEOWNERS

```
@@ -315,7 +326,22 @@
 /packages/system @elastic/obs-infraobs-integrations
 /packages/system/changelog.yml @elastic/obs-infraobs-integrations @elastic/sec-linux-platform @elastic/sec-windows-platform
 /packages/system/data_stream/auth @elastic/sec-windows-platform
-/packages/system/data_stream/security @elastic/sec-linux-platform @elastic/sec-windows-platform
+/packages/system/data_stream/security @elastic/sec-windows-platform
+/packages/system/data_stream/application @elastic/obs-infraobs-integrations
```

```suggestion
/packages/system/data_stream/application @elastic/sec-windows-platform
```

because this is collecting windows event logs using the winlog input owned by that team.

---

ok makes sense, thx for catching that 🙂